src/App.css

```diff
@@ -19,6 +19,35 @@
   color: white;
 }
 
+.user-panel {
+  height: 100%;
+  width: 100%;
+}
+
+.user-photo {
+  top:0;
+  right: 2em;
+  border-radius: 50%;
+  padding: 1em;
+  height: 100px;
+  width: 100px;  
+}
+
+.App-icon {
+  height: 64px;
+  width: 64px;  
+}
+
+.login-button {
+ margin-left: 1em; 
+ background: transparent;
+ border: 0;
+}
+
+.minitext {
```

Comment:
maybe this should be exported in another separate component for this as well, maybe a  <Text size="small"> component would be cool for this